### ISSUE TITLE
NewContextMenu() : Adjust position to ensure visibility

### ISSUE THREAD

**Issue by beanpole135** [maxence-charriere/go-app#507]
Module Version Tested: 	`github.com/maxence-charriere/go-app/v7 v7.2.0`

Problem: When the button that requests a context menu is near the right-hand side of the screen, it will "overflow" outside the bounds of the window and prevent parts of the menu from being visible.

Example Screenshot:
![image](https://user-images.githubusercontent.com/4172864/110360777-29d9c580-800d-11eb-9403-669971afe7a4.png)

Solution: go-app appears to be manually setting the "top" and "left" object properties based on mouse position. This just needs to check/adjust the value based on the window size and relative position:

Solution pseudo-code:
```
if (menu.X + menu.Width) > window.Width { menu.X = window.Width - menu.Width }
if (menu.Y + menu.Height) > window.Height { menu.Y = window.Height - menu.Height }
if menu.Y < 0 { menu.Y = 0 } //never go above top of screen
//Then set menu top (Y) and left (X) properties with these adjusted values
```

**Comment by beanpole135** [comment]
Update: I just noticed that version 7.3.0 was available. I updated the module and rebuild my site with that version and the issue still persists.

I also browsed through the go-app sources and found where my "pseudo-code" solution was already in-place right now (https://github.com/maxence-charriere/go-app/blob/19c26e74e0ecf050a81e9783e7781fc269ba6b7e/pkg/app/contextmenu.go#L196), so the only option I can think of that might be causing the issue is if one of the "width" measurements in the code is not accurate right now.
